Objects/setobject.c

```diff
@@ -935,8 +956,19 @@ set_update_internal(PySetObject *so, PyObject *other)
     return 0;
 }
 
+/*[clinic input]
+set.update
+
+    so: setobject
+    *args: object
```

Comment:
Let's try and keep the existing Python parameter names intact. You can use the `as` syntax for this:

```suggestion
    *others as args: object
```